include/boost/hana/traits.hpp

```diff
@@ -67,7 +67,9 @@ BOOST_HANA_NAMESPACE_BEGIN namespace traits {
     constexpr auto is_trivial = detail::hana_trait<std::is_trivial>{};
     constexpr auto is_trivially_copyable = detail::hana_trait<std::is_trivially_copyable>{};
     constexpr auto is_standard_layout = detail::hana_trait<std::is_standard_layout>{};
+#if __cplusplus < 202002L || defined(BOOST_HANA_CONFIG_ENABLE_DEPRECATED_IS_POD)
```

Comment:
I think I'd just drop it altogether. Being Boost and not the standard, I think it's more reasonable to be slightly more aggressive about these things. WDYT?